src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -31,7 +31,7 @@ public void testCallCheck() {
 	public void testIsFull() {
 		DefaultCountingOutRhymer ryhmer = new DefaultCountingOutRhymer();
 		final int STACK_CAPACITY = 12;
-		for (int i = 0; i < STACK_CAPACITY; i++) {
+		for (int i = 1; i < STACK_CAPACITY; i++) {
```

Comment:
Is it a correct approach to fixing errors showed by tests?
Errors are still within the project. STACK_CAPACITY is 12 so 12 elements should be added to the stack.